migration/migrator/triggers/master/sync_courses_user.sql

```diff
@@ -0,0 +1,79 @@
+--
+-- Name: sync_courses_user(); Type: FUNCTION; Schema: public; Owner: -
+--
+
+CREATE OR REPLACE FUNCTION public.sync_courses_user() RETURNS trigger
+    LANGUAGE plpgsql
+    AS $$
+            DECLARE
+                user_row record;
+                db_conn varchar;
+                query_string text;
+            BEGIN
+                db_conn := format('dbname=submitty_%s_%s', NEW.semester, NEW.course);
+
+                IF (TG_OP = 'INSERT') THEN
+                    -- FULL data sync on INSERT of a new user record.
+                    SELECT * INTO user_row FROM users WHERE user_id=NEW.user_id;
+                    query_string := 'INSERT INTO users (
+                        user_id,
+                        user_pronouns,
```

Comment:
for order consistency pronouns & numeric_id should be swapped
(they are swapped in the sync_user.sql trigger function)